camilo-alfonso/final-project/src/components/Crew/Crew.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from "react";
+import axios from "axios";
+import { useDispatch } from "react-redux";
+import { setCrew } from "../../redux/actions/crewActions";
+import { SPACE_API } from "../../apis/apiLink";
+import { Box, VStack, Heading } from "@chakra-ui/react";
+import CrewList from "./CrewList";
+
+const Crew = () => {
+  const dispatch = useDispatch();
+
+  const fetchAstronauts = async () => {
```

Comment:
@camosu Since you are using redux all data consumption is consider a side effect, so you should thunk for API call and data logic

Reply:
Hi @emolinaglobant,  I have removed the commented code and moved the API requests logic out of the presentational components.